migrate-tags.js

```diff
@@ -0,0 +1,136 @@
+/*
+This is a one-off script to add type prefixes to object keys that don't have one.
+*/
+
+const process = require("process");
+const args = require("commander");
+const { Client } = require("pg");
+const winston = require("winston");
+const config = require("./config");
+const moment = require("moment");
+
+const MIN_CONFIDENCE = 0.6;
+const animals = [
+  "possum",
+  "rodent",
+  "mustelid",
+  "hedgehog",
+  "cat",
+  "bird",
+  "kiwi",
+  "dog",
+  "leporidae",
+  "human",
+  "insect",
+  "pest",
+  "part"
+];
+Object.freeze(animals);
+
+async function main() {
+  args
+    .option("--config <path>", "Configuration file", "./config/app.js")
+    .parse(process.argv);
+
+  config.loadConfig(args.config);
+
+  const pgClient = await pgConnect();
+  const recordings = await getSingleTrackRecordings(pgClient);
+  let manualAnimals;
+  logger.info(`Found ${recordings.length} possible recordings`);
+  for (const [rId, trackId, oldTags] of recordings) {
+    manualAnimals = getManualAnimalTags(oldTags);
```

Comment:
Why not just `const manualAnimal = ...`? No need to define this outside of the loop.